test/sequential/test-heapsnapshot-near-heap-limit.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
Should this be a pummel test instead?

Reply:
The allocation done here is fairly small (in the size of 10MB to 20MB). I think it would be good to have some tests for smaller heaps in sequential (hm, though on a second thought, I don't see why they can't be in parallel?) and have a test for bigger heaps in pummel?

Reply:
I added a test for a heap with 512MB in pummel (it could be bigger, but will also take more time to complete)